src/uct/tcp/tcp.h

```diff
@@ -291,6 +291,7 @@ typedef struct uct_tcp_iface {
                                                       * are in progress + how many EPs are
                                                       * waiting for PUT Zcopy operation ACKs
                                                       * (0/1 for each EP) */
+    ucs_range_spec_t              ports_range;       /** Range of ports to use for bind() */
```

Comment:
ports_range -> port_range in all places
"/proc/sys/net/ipv4/ip_local_**port**_range"